### ISSUE TITLE
Site request: Libgen (Comics)

### ISSUE THREAD

**Issue by Eli2712** [manga-download/hakuneko#1519]
HakuNeko has lots of manga, however, the there are not many comics.
Can you add the website https://libgen.lc/comics/ into HakuNeko, from there it could be organized.  Here are many comics that you could add/organize:

Adam
Agnes
Amazing Spider-man
Andy Capp
Argyle Sweater
Ask Shagg
B.C.
Baby Blues
Baldo
Barney & Clyde
Barney Google
Beetle Bailey
Big Nate
Bizarro
Blondie
Bo Nanas
Boondocks
Brewster Rockit
Brevity
Broom Hilda
Candorville
Cathy
Close to Home
Cornered
Curtis
Dennis the Menace
Diamond Lil
Dilbert
Doonesbury
F Minus
Flying McCoys
For Better or For Worse
Foxtrot
Frank & Ernest
Frazz
Fusco Brothers
Garfield
Get Fuzzy
Hagar the Horrible
Hi and Lois
In the Bleachers
Judge Parker
Liberty Meadows
Lio
The Lockhorns
Long Story Short
Loose Parts
Mark Trail
Marvin
Mary Worth
Mike du Jour
Momma
Mutts
Nancy
On the Fastrack
Non Sequitur
One Big Happy
Overboard
Peanuts
Pearls Before Swine
The Phantom
Pickles
Prickly City
Real Life Adventures
Red and Rover
Reply All
Rex Morgan, M.D.
Richard’s Poor Almanac
Rudy Park
Sally Forth
Sherman's Lagoon
Shoe
Single and Looking
Six Chix
Slylock Fox
Speed Bump
Stone Soup
Tank McNamara
Tom the Dancing Bug
The Wizard of Id
WuMo
Zippy the Pinhead
Zits

**Comment by Sheepux** [comment]
The website itself already provides cbr archives. Unsure if it should be included in hakuneko.

**Comment by XenonR** [comment]
Although i love the idea of adding Lib Genesis i also think it doesn't make much sense. You can easily download (and extract) what you need. Additionaly this would only be an URI handler since i can see no index and the search requires minimum 3 letters. Which would result in at least **2541 billion queries** to build an index.

**Comment by Sheepux** [comment]
How about adding the connector but simply giving a popup "you can already download from the website. Just go to xyz.url"

**Comment by XenonR** [comment]
The more I think about it I feel a normal download manager is better suited for this website.

**Comment by 27136773** [comment]
I cannot reach the website is this region locked?
also adding a website where chapters can be downloaded and links are not hidden behind an ad wall, i see no point in adding this...

**Comment by Sheepux** [comment]
That's why i proposed to implement it as a warning stating that you cannot do anything within hakuneko (so that nobody requests it again).

/agree with closing though

**Comment by XenonR** [comment]
Comics are ready dowloadable as CBZ without any ad wall.
/Closed

**Comment by 27136773** [comment]
The problem with popups is, that users are probably clicking them away instantly pretending nothing bad happened...
Think of the `The website does not provide a manga list, use copy & paste` popup in relation to the discord questions why there is no manga list 😁
